assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -130,24 +136,42 @@ function InnerElement({
   const posterSrc = type === ContentType.GIF ? output.poster : poster;
   const displayPoster = posterSrc ?? newVideoPosterRef.current;
 
+  // Get the same size for the clone as it will be when inserted.
```

Comment:
@barklund Actually this part here might still be relevant -- it will match the clone size with the size of just clicking an image of the library. Currently, when clicking on media, it will result in different sized element compared to when dragging media. Should we create a separate issue for that?